### ISSUE TITLE
Some issues I found #1

### ISSUE THREAD

**Issue by cheesemanhugs** [Althro/Tinvaak#13]
EDIT: Also noticed that the flame atronach spell doesn't progress the first college of winterhold quest to get into the college. I'll cast it on the designated area and nothing happens. The quick fix is reloading and getting a different spell.

I've played for a while and as promised will post some bugs/oddities I noticed.

Location: West whiterun hold - barley dark border

Bug (lod issue): Not sure if this is a bug really but the lod doesn't contain the door for this gate so its just empty. Also one time the gate was there in the distance but it was completely white (This was at night)

![The Elder Scrolls V  Skyrim Special Edition Screenshot 2021 05 03 - 19 23 42 74](https://user-images.githubusercontent.com/64744940/118211336-04689e00-b43a-11eb-836c-a054e08debd2.png)
![The Elder Scrolls V  Skyrim Special Edition Screenshot 2021 05 03 - 19 34 26 87](https://user-images.githubusercontent.com/64744940/118211672-793bd800-b43a-11eb-833a-56a6d90da005.png)

-------------
Mod Culprits: Distinct interiors 
Bug: There's a chair underneath this longtable so it causes this visual bug when someone sits in it.
![The Elder Scrolls V  Skyrim Special Edition Screenshot 2021 05 03 - 19 28 53 36](https://user-images.githubusercontent.com/64744940/118211494-324de280-b43a-11eb-90c0-94ef26431eb9.png)

------------
Mod Culprits: Distinct interiors
Location: Hall of the dead (interior)
Bug:  Similar bug here with this eagle object. When an npc sits in this chair they clip through the eagle mesh
![The Elder Scrolls V  Skyrim Special Edition Screenshot 2021 05 03 - 19 30 55 71](https://user-images.githubusercontent.com/64744940/118211531-42fe5880-b43a-11eb-8260-2305d540d18b.png)

---------------------------------

Bug: this cooking pot and piece of wall are floating here. They appear when you get closer
Location: Above bleakwind basin (to the left of whiterun)
![The Elder Scrolls V  Skyrim Special Edition Screenshot 2021 05 03 - 19 37 39 10](https://user-images.githubusercontent.com/64744940/118212041-a6888600-b43a-11eb-903a-1bd30d7ccce3.png)
![The Elder Scrolls V  Skyrim Special Edition Screenshot 2021 05 03 - 19 38 10 60](https://user-images.githubusercontent.com/64744940/118212233-b2744800-b43a-11eb-88cb-6dd2bd55cb22.png)
--------------------------------
Bug: Hoth seems a little bugged. Moves his mouth constantly with no words coming out. Some of his voice lines don't play (Or maybe he just doesn't have any for certain parts?). And he acts like we've known each other on my first time meeting him. This could all be how the mod is (Never used hoth in a playthrough). Either way its jarring and buggy.
------------------------------------
Bug: Flickery interior lighting at these steps
Location: Nightcaller temple (Dawnstar)
![The Elder Scrolls V  Skyrim Special Edition Screenshot 2021 05 12 - 02 44 56 61](https://user-images.githubusercontent.com/64744940/118212878-de8fc900-b43a-11eb-8a3c-167e9aa69f35.png)

Bug: Same issue in highmoon hall (morthal)
![The Elder Scrolls V  Skyrim Special Edition Screenshot 2021 05 13 - 20 40 06 07](https://user-images.githubusercontent.com/64744940/118212919-f6ffe380-b43a-11eb-9381-6a6714f56aa6.png)
![The Elder Scrolls V  Skyrim Special Edition Screenshot 2021 05 13 - 20 40 11 44](https://user-images.githubusercontent.com/64744940/118212921-f8c9a700-b43a-11eb-8afd-29736205b220.png)
-------------------------------------
bug: no footstep/splashy sounds on this water/rock path (I'm sure all similar rock/water paths have the same issue)
Location: Dimhollow crypt (Dawnguard DLC)
![The Elder Scrolls V  Skyrim Special Edition Screenshot 2021 05 12 - 09 34 24 92](https://user-images.githubusercontent.com/64744940/118212977-1434b200-b43b-11eb-997a-017fc8bb9d29.png)

-----------------------
Question:
While playing a vampire with sunhelm the only need I have to worry about is thirst. It never goes up though. I'm guessing this is intended since the sacrilege vampire overhaul mod adds its own hunger system, but sunhelm has a problem where if you pass time by waiting your thirst will still go up giving you the debuff. It goes away by itself a couple minutes after but I was just curious if that's intended or some kind of bug/conflict because it does feel a little odd.

**Comment by Althro** [comment]
Flame Atronach is a vanilla bug and is known. Not much I can do to fix that sadly.

Lod issue: Fixed in new version. I've swapped to the newer dyndolod which will remedy issue like this.

Distinct Interiors: These will all be fixed in the new version. I missed some patches in 1.x and have been redoing towns and cities for 2.0 to ensure compat with distinct interiors.

Cooking pot and wall: Known issue. Looking into fixing.

Hoth: This is as he is iirc. I'll double check him, but from memory that sounds like him so not a bug.

Lighting: Fixed in 2.0. I've redone lighting completely for the next version so issues like this will hopefully not come up.

Sounds: This is not something I have ever been able to replicate on my end. All sounds work fine for me. If you're having multiple issue with sounds, try the solution I provided in #12. 

Sunhelm: This should be fixed in 2.0 as I've updated to Sunhelm 2.x.
